powerauth-restful-security-javaee/src/main/java/io/getlime/security/powerauth/rest/api/jaxrs/service/v3/TokenService.java

```diff
@@ -128,8 +128,8 @@ public TokenRemoveResponse removeToken(TokenRemoveRequest request, PowerAuthApiA
             response.setTokenId(tokenId);
             return response;
         } catch (Exception ex) {
-            logger.warn("Removing PowerAuth token failed", ex);
-            throw new PowerAuthAuthenticationException(ex.getMessage());
+            logger.warn("Removing PowerAuth token failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.